specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -94,11 +95,26 @@ Cells are the smallest unit of blob data that can come with their own KZG proofs
 #### `bytes_to_cell`
 
 ```python
-def bytes_to_cell(cell_bytes: Vector[Bytes32, FIELD_ELEMENTS_PER_CELL]) -> Cell:
+def bytes_to_cell(cell_bytes: CellBytes) -> Cell:
     """
     Convert untrusted bytes into a Cell.
     """
-    return [bytes_to_bls_field(element) for element in cell_bytes]
+    cell = Cell()
+    for i in range(FIELD_ELEMENTS_PER_CELL):
+        value = bytes_to_bls_field(cell_bytes[i * BYTES_PER_FIELD_ELEMENT: (i + 1) * BYTES_PER_FIELD_ELEMENT])
+        cell[i] = value
+    return cell
+```
+
+```python
+def cell_to_bytes(cell: Cell) -> CellBytes:
+    """
+    Convert a Cell into bytes.
+    """
+    cell_bytes = b""
```

Comment:
curious if `b""` is the common way to do this and or whether `cell_bytes = []` would work